### PR TITLE
adding poem_sentiment

### PR DESCRIPTION
**Pull request by patil-suraj**
Adding poem_sentiment dataset.
https://github.com/google-research-datasets/poem-sentiment

### CODE REVIEW

File: datasets/poem_sentiment/poem_sentiment.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        ]
+
+    def _generate_examples(self, filepath):
+        with open(filepath, encoding="utf-8") as f:
+            lines = f.readlines()
+            for line in lines:
+                fields = line.strip().split("\t")
+                idx, verse_text, label = fields
+                label = int(label)
+                label = label + 1 if label == 1 or label == -1 else label
```
Comment (lhoestq) [inline datasets/poem_sentiment/poem_sentiment.py:84]: is it possible to end up with label=1 for positive this way ??
label = -1 -> 0
label = 0 -> 0
label = 1 -> 2

**Review by lhoestq — COMMENTED** [review]
Nice :) 

I left one comment about the labels, let me know what you think

File: datasets/poem_sentiment/poem_sentiment.py (hunk 1 above)
Comment (patil-suraj) [reply datasets/poem_sentiment/poem_sentiment.py]: My bad. And thanks Quentin, while fixing this, I also realised that there is another label "mixed" which I missed to include

**Review by lhoestq — APPROVED** [review]
Nice thank you !

**Pull request merged**